lib/input.rb

```diff
@@ -0,0 +1,11 @@
+class Input
```

Comment:
This class is really a test-only class. If I see it alongside the product code, I am likely to be confused. I would put it alongside (or even inside) the test that uses it.